instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/util.py

```diff
@@ -68,3 +68,16 @@ def _format_command_args(args):
         out_str = ""
 
     return out_str
+
+
+def _set_span_attribute(span, name, value):
```

Comment:
Maybe call this `_set_span_attribute_if_set` to hint that there is a filter?

Reply:
I changed it to `_set_span_attribute_if_value`